include/Runtime/platform.h

```diff
@@ -112,6 +112,16 @@ typedef void (*Event)(void);
 #endif
 #endif
 
+#ifndef ATTRIBUTE_NORETURN
+#if defined(__MWERKS__) || defined(__GNUC__)
+#define ATTRIBUTE_NORETURN
+#elif defined(__clang__)
+#define ATTRIBUTE_NORETURN __attribute__((noreturn))
+#else
+#error unknown compiler
```

Comment:
Let's not error here, the fallback should be to define it to nothing and continue